_posts/2021-04-19-Princeton Departments by Distribution.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+author: José Pablo Fernández García
```

Comment:
This is nit-picky, but nevertheless important: in your YAML header (that's what this is at the top of every page) some of these categories are case sensitive because they refer to either folders or Liquid tags we've created elsewhere in the site. In this case, author needs to be "author" not "Author" since because we've added code to tell Jekyll to search a directory called "people" for names that match the category "author," which are then fed into a layout called "author." I've gone ahead and changed it for you, but I wanted you to know in future for your final project.